### DISCUSSION TITLE
请问可以自己添加法语的听写内容，然后自动纠正法语吗？

### DISCUSSION THREAD

**Discussion by lishan256** [ZuodaoTech/everyone-can-use-english#546] [Ideas]
老师好，我目前法语提高比英语要迫切，想知道enjoy是否可以对法语提供跟英语同样的支持？只需要我自己上传相关资料，就可以了吗？

**Comment by an-lee** [comment]
这个需求可以自己用智能助手实现，新建一个对话，选自定义角色，然后参考一下“英语教练”的角色定义写一个法语版本的就可以了。

**Reply by lishan256** [reply]
我没有说清楚我的需求，抱歉。您说得这部分，chatgpt现在就可以做到，但是语音对比，chatgpt没有。我主要是想纠正法语的发音。

**Reply by ksdaylight** [reply]
你好，貌似是他这样说的，英标和发音评估貌似都是用的英语的接口

**Reply by an-lee** [reply]
后续会把其他语言支持上

**Comment by an-lee** [comment]
v0.2.9 版本以后，支持设置多种“学习语言”了，支持法语。发音评估这些也都支持

**Reply by lishan256** [reply]
非常感谢！

**Reply by lishan256** [reply]
用起来体验不错，非常感谢！
